lib/githubbot.ts

```diff
@@ -251,7 +252,7 @@ export class GithubBot extends ProcBot.ProcBot<string> {
                         // If there's more retries, try again in 5 seconds.
                         setTimeout(runApi, 5000);
                     }
-                }).then(resolve);
+                });
             };
```

Comment:
Sure, yeah, that bug definitely needed fixing (:+1:), but even long-term making this clear and readable is the important bit, async/await or otherwise.

I'll take this moment to plug this great intro to async/await if you want more background: https://www.youtube.com/watch?v=Hn-gkM0ARZU :smile:

Or Jake Archibald has a great guide (as ever) which might be a more efficient route: https://developers.google.com/web/fundamentals/getting-started/primers/async-functions